packages/ui-utils/src/hash.ts

```diff
@@ -52,11 +50,10 @@ function toBase64(input) {
   return output
 }
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'input' implicitly has an 'any' type.
-function executeHash(input) {
+function executeHash(input: string): string {
   let hash = 0
 
-  if (input.length === 0) return hash
+  if (input.length === 0) return '0'
```

Comment:
I this case it was returning a number (whereas in every other case a string), I've fixed it